src/components/Note/Note.scss

```diff
@@ -58,7 +58,9 @@ $note__author-image-size: 26px;
 }
 
 .note:focus-within,
-.note:focus,
+.note:focus {
+  outline: 2px solid rgba(var(--accent-color-rgb), 0.5);
```

Comment:
full opacity looks better imo, because it matches the small colored strip on the left side of the cards. What do you think?